.gitignore

```
@@ -112,4 +112,7 @@ buildNumber.properties
 **/*.lock
 **/*.tgz
 
+# Generated bin folders
+**/bin/
```

```suggestion
bin/
```

this is enough to tell "all the bin folders in root and all the subfolders"